packages/vm/.eslintrc.js

```diff
@@ -1,18 +1,10 @@
 module.exports = {
   extends: "@ethereumjs/eslint-config-defaults",
-  ignorePatterns: ["tests", "scripts", "examples", "karma.conf.js"],
+  ignorePatterns: ["scripts", "examples", "karma.conf.js"],
   rules: {
-    '@typescript-eslint/await-thenable': 'off',
-    '@typescript-eslint/no-floating-promises': 'off',
-    '@typescript-eslint/no-unused-vars': 'off',
     '@typescript-eslint/no-use-before-define': 'off',
-    '@typescript-eslint/restrict-plus-operands': 'off',
-    'no-async-promise-executor': 'off',
     'no-invalid-this': 'off',
     'no-restricted-syntax': 'off',
```

Comment:
This rule disables syntax like `for(x in y)`, which I think is too strict.

Reply:
Yeah, I would also say `for (x in y)` restriction is too strict, it might generally be useful for code readability to decide on some obscure or seldomly used language features to disallow, e.g. this `with()` statement (never heard of that one) mentioned in the [docs](https://eslint.org/docs/rules/no-restricted-syntax) seems to be some useful case I would say (but generally: rather as an example here, haven't thought about this that much yet).

I didn't find what rules ARE actually activated by default on ESLint, does anyone has a link here? 🤔

Reply:
It seems to be:

`"no-restricted-syntax": ["error", "FunctionExpression", "WithStatement", "BinaryExpression[operator='in']"]`

We might take a look at this one again in a follow-up PR, since this seems a rather "big" ignore which we are disabling.